documentation/datastructure.dot

```diff
@@ -0,0 +1,210 @@
+    // -------
+    hasName [label="hasName\nString"];
+    Project -> hasName [label="1"];
+
+    hasDescription [label="hasDescription\nString"];
+    Project -> hasDescription [label="1"];
+
+    hasKeywords [label="hasKeywords\nString"];
+    Project -> hasKeywords [label="1-n"];
+
+    hasDiscipline [label="hasDiscipline\nString | URL"];
+    Project -> hasDiscipline [label="1-n"];
+
+    hasStartDate [label="hasStartDate\nDate"];
+    Project -> hasStartDate [label="1"];
+
+    hasEndDate [label="hasEndDate\nDate"];
+    Project -> hasEndDate [label="0-1"];
+
+    hasTemporalCoverage [label="hasTemporalCoverage\nString | URL"];
+    Project -> hasTemporalCoverage [label="1-n"];
+
+    Project -> hasSpatialCoverage [label="1-n"];
+    hasSpatialCoverage -> Place
+
+    Project -> hasFunder [label="1-n"];
+    hasFunder -> Person;
+    hasFunder -> Organization;
+
+    Project -> hasGrant [label="0-n"];
+    hasGrant -> Grant;
+
+    hasURL [label="hasURL\nURL"];
+    Project -> hasURL [label="1-2"];
+
+    hasShortcode [label="hasShortcode\nString"];
+    Project -> hasShortcode [label="1"];
+
+    hasAlternateName [label="hasAlternateName\nString"];
+    Project -> hasAlternateName [label="0-n"];
+
+    Project -> hasDataManagementPlan [label="0-1"];
+    hasDataManagementPlan -> DataManagementPlan;
+
+    hasPublication [label="hasPublication\nString"];
+    Project -> hasPublication [label="0-n"];
+
+    Project -> hasContactPoint [label="0-1"];
+    hasContactPoint -> Person;
+    hasContactPoint -> Organization;
+
+
+    // Person
+    // ------
+    sameAs2 [label="sameAs\nURL"];
+    Person -> sameAs2 [label="0-n"];
+
+    hasGivenName [label="hasGivenName\nString"];
+    Person -> hasGivenName [label="1-n"];
+
+    hasFamilyName [label="hasFamilyName\nString"];
+    Person -> hasFamilyName [label="1-n"];
+
+    hasEmail [label="hasEmail\nIRI"];
+    Person -> hasEmail [label="0-2"];
+
+    Person -> hasAddress [label="0-n"];
+    hasAddress -> PostalAddress
+
+    Person -> isMemberOf [label="1-n"];
+    isMemberOf -> Organization
+
+    hasJobTitle [label="hasJobTitle\nString"];
+    Person -> hasJobTitle [label="1-n"];
+
+    hasRole [label="hasRole\nString"];
+    Person -> hasRole [label="1-n"];
+
+
+    // Organization
+    // ------------
+    hasName2 [label="hasName\nString"];
+    Organization -> hasName2 [label="1-n"];
+
+    hasEmail2 [label="hasEmail\nIRI"];
+    Organization -> hasEmail2 [label="0-1"];
+
+    hasAddress2 [label="hasAddress"];
+    Organization -> hasAddress2 [label="0-1"];
+    hasAddress2 -> PostalAddress;
+
+    hasURL2 [label="hasURL\nURL"];
+    Organization -> hasURL2 [label="0-1"];
+
+
+    // Grant
+    // -----
+    hasName3 [label="hasName\nString"];
+    Grant -> hasName3 [label="0-1"];
+
+    hasURL3 [label="hasURL\nURL"];
+    Grant -> hasURL3 [label="0-1"];
+
+    hasNumber [label="hasNumber\nString"];
+    Grant -> hasNumber [label="0-1"]
+
+    hasFunder [label="hasFunder\nString"];
```

Comment:
Funder can be either `Person` or `Organization` (lines 119-121), so to be consistent, type should be removed.

Reply:
Correct. Fixed.